server/src/controllers/customers.controller.ts

```diff
@@ -73,9 +69,10 @@ export class CustomerController {
   async find(
     @param.filter(Customer) filter?: Filter<Customer>,
   ): Promise<Customer[]> {
-    return this.customerRepository.find(filter);
+    return this.customerRepository.find({include: ['users']});
```

Comment:
why not using filter here?

Reply:
use the filter instead of directly include